set.mm

```diff
@@ -434160,8 +434159,7 @@ orthogonal vectors (i.e. whose inner product is 0) is the sum of the
 */
 htmldef "^" as "<IMG SRC='uparrow.gif' WIDTH=7 HEIGHT=19 ALT=' ^' TITLE='^'>";
   althtmldef "^" as '&uarr;';
-  latexdef "^" as "\uparrow"; /*
-  latexdef "^" as "\widehat{\ }"; */
+  latexdef "^" as "\uparrow";
```

Comment:
same in `iset.mm`

Reply:
ping @david-a-wheeler, and after we can merge

Reply:
Okay, I'll make that change as well!